chains/chains_dev.json

```diff
@@ -1,96 +1,96 @@
 [{
-        "disabled": false,
-        "chainId": "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3",
```

Comment:
We don't see on `chains/chains_dev.json` now. We see on `chains/v1/chains_dev.json` and agreed to change only last version of file. Let's change `chains_dev.json` in `v1` folder